simulation/include/pcl/simulation/camera.h

```diff
@@ -5,14 +5,15 @@
 #include <boost/shared_ptr.hpp>
 
 #include <pcl/eigen_base.h>
+#include <pcl/pcl_exports.h>
 
 namespace pcl {
 namespace simulation {
 
 class PCL_EXPORTS Camera {
 public:
-  using Ptr = shared_ptr<Camera>;
-  using ConstPtr = shared_ptr<const Camera>;
+  using Ptr = boost::shared_ptr<Camera>;
```

Comment:
It was intentional that `shared_ptr` is used without namespace. Before your previous commit, this file included "pcl_macros.h" header which provided `pcl::shared_ptr`.

Reply:
One point in favour of an eventual memory.h made out of shared_ptr.h and make_shared.h.